src/Util/Search.php

```diff
@@ -139,6 +139,30 @@ public function buildQuery(array $requestTerms): Query
         // Bool query to get range temporal extent dates
         $collectionDateBoolQuery = new Query\BoolQuery();
 
+        // Check if exclude term exists in the given query term
+        $mustNotQueryTerm = '';
+        if (preg_match_all('/-\b(\w*\w*)\b/', $queryTerm, $matches)) {
```

Comment:
Only one `\w` seems to word too. And one could instead of the last word boundry search for `\S*` (any space/new line etc). (optional)